packages/@o3r/eslint-config/src/rules/typescript/eslint.cjs

```diff
@@ -12,7 +12,6 @@ const config = [
     ],
     rules: {
       ...convertWarningsToErrors(eslint.configs.recommended),
-      'camelcase': 'error',
```

Comment:
do we want to disable this rule for imports?

Reply:
This rule doesn't trigger issue on imports by default because it is the following 
`{
    selector: 'import',
    format: ['camelCase', 'PascalCase'],
  }`